Sources/Nuke/Decoding/AssetType.swift

```diff
@@ -28,6 +28,8 @@ public struct AssetType: ExpressibleByStringLiteral, Hashable, Sendable {
     /// iOS 14, watchOS 7, tvOS 14.
     public static let webp: AssetType = "public.webp"
 
+    public static let animatedWebp: AssetType = "public.animated.webp"
```

Comment:
I don't think `"public.animated.webp"` is a thing.

I suggest adding something similar to [`ImageProperties`](https://github.com/kean/Nuke/blob/22ab51a030517fe2e170c4569b98d47fac4087a5/Sources/Nuke/Decoding/ImageDecoders%2BDefault.swift#L177) and potentially making the existing type public. It's currently internal because one of Nuke's goal is to keep the public API small.